docs/configuration/integrations/bitbucket-server.md

```diff
@@ -26,3 +26,15 @@ Follow the instructions below to set up the Codacy integration with Bitbucket Se
     ```
 
 After this is done you will be able to use Bitbucket Server to authenticate to Codacy.
+
+## Troubleshooting
+
+- During an authentication procedure, if you got stuck on the provider and this message is shown
+  - ![Invalid client id](./bitbucket-invalid-client-id.png)
+    It means you have not introduced the client id when configuring Bitbucket.com on Codacy.
+    - Make sure the value matches the one in your Bitbucket.com application
+    - If you still could not find the problem:
+        - Extract the parameter `client_id`, from the browser address bar in Bitbucket.com where the error appears (e.g.: `r8QJDkkxj8unYfg4Bd`)
```

Comment:
```suggestion
        - Extract the parameter `client_id`, from the browser address bar in your Bitbucket Server URL where the error appears (e.g.: `r8QJDkkxj8unYfg4Bd`)
```

Reply:
BB Server was not supposed to be committed. All the errors will be handled in our side.